front-end/src/components/Login/Forgot/index.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import './index.scss';
+import $ from 'jquery';
+
+class Forgot extends Component{
+
+    componentDidMount = () => {
+        $('.forgot__close').on('click', function () {
```

Comment:
Please check tab settings in your IDE. Tab should be 2 spaces, not 4, according to the ESLint settings.
![image](https://user-images.githubusercontent.com/10584616/47606480-baa33e00-da1c-11e8-9f77-1f45c86e4dc0.png)

You should also configure ESLint plugin to enforce the rules in .eslint. Run 'npm install' first, I've added needed dependencies for ESlint to the package.json on this branch
![eslint-config](https://user-images.githubusercontent.com/10584616/47606518-1d94d500-da1d-11e8-8777-8becb37bc1f7.png)